docs/docs/handling-events.md

```diff
@@ -2,7 +2,113 @@
 id: handling-events
 title: Handling Events
 permalink: docs/handling-events.html
-prev: state-and-lifecycle.html
 ---
 
-TODO
+Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+
+* React events are named using camelCase, rather than lowercase.
+* With JSX you pass a function as the event handler, rather than a string.
+
+For example, the HTML:
+
+```html
+<button onclick="activateLasers()">Activate Lasers</button>
+```
+
+is slightly different in React:
+
+```js
+<button onClick={activateLasers}>Activate Lasers</button>
+```
+
+Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default link behavior of opening a new page, you can write:
+
+```html
+<a href="#" onclick="console.log('the link was clicked'); return false">
+  click me
+</a>
+```
+
+In React, this could instead be:
+
+```js
+function ActionLink() {
+  const handleClick = (e) => {
```

Comment:
Any specific reason for using an arrow here? Seems like would be consistent to use vanilla function like the outer one.